apps/antalmanac/src/components/Calendar/Toolbar/EditSchedule/ScheduleNameDialog.tsx

```diff
@@ -13,6 +12,7 @@ import {
 import { withStyles } from '@material-ui/core/styles';
 import { ClassNameMap } from '@material-ui/core/styles/withStyles';
 import { Add, Edit } from '@material-ui/icons';
+import { forwardRef, useCallback, useState, useMemo, MouseEvent, ChangeEvent, KeyboardEvent } from 'react';
```

Comment:
I'll recap this in the overall review as well, but please distinguish between type imports and named imports whenever possible.

```suggestion
import { forwardRef, useCallback, useState, useMemo, type MouseEvent, type ChangeEvent, type KeyboardEvent } from 'react';
```